src/eCloud/ApplianceVersionClient.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace UKFast\SDK\eCloud;
+
+use UKFast\SDK\eCloud\Entities\Appliance;
+use UKFast\SDK\eCloud\Entities\Appliance\Version\Data;
+use UKFast\SDK\Page;
+
+class ApplianceVersionClient extends Client
```

Comment:
I don't think this file is necessary. We have the data method in the ApplianceClient to surface appliance version data to customers, and we're not surfacing appliance-version endpoints to customers at this time. appliance-version endpoints will also be admin only so would/should not work for customers.